modules/minimap2/1.0/envs/minimap2-2.24.yaml

```diff
@@ -0,0 +1 @@
+/projects/rmorin/projects/gambl-repos/gambl-hshaalan/src/lcr-modules/envs/minimap2/minimap2-2.24.yaml
```

Comment:
Can you please update the symlink to point to the `lcr-modules/envs/minimap2` directory to parallel the samtools symlink below?